src/parser.js

```diff
@@ -669,7 +689,7 @@ class TokenParser {
 
         case 'LITERAL':
           if (this.currentNode.started) {
-            if (chr === '\u0000') {
+            if (chr === 0) {
```

Comment:
I believe the unicode for 0 is U0030. U0000 is the NUL character, I suppose, in that case, chr would be null here..?

Reply:
I am checking for the null character here, not '0'. chr can have values 0-255 and the charCode for the null character is 0.